src/hal/src/image.rs

```diff
@@ -396,7 +400,7 @@ impl Into<f32> for Lod {
     }
 }
 
-/// A wrapper for the 8bpp RGBA color, encoded as u32.
+/// A wrapper for an RGBA color with 8 bits per pixel, encoded as a u32.
```

Comment:
in the long run, we should tighten up the use of "pixel" versus "texel" versus "fragment".

Reply:
Which is it in this case?  Texel?

Reply:
AFAIK, this is the distinction:
  - "texel" refers to the actual contents of an image, which is the case here
  - "pixel" refers to the *result* on screen
  - "fragment" refers to the unit of work for the shader, multiple fragments can contribute to the same pixels